src/integrationTest/java/com/mongodb/hibernate/BasicCrudIntegrationTests.java

```diff
@@ -321,6 +316,11 @@ static class Book {
     static class BookCountByAuthor {
         String _id;
         int count;
+
+        public BookCountByAuthor(String _id, int count) {
+            this._id = _id;
+            this.count = count;
+        }
```

Comment:
this constructor is required to create ResultClass instance based on select fields.